### DISCUSSION TITLE
Current Media playing widget

### DISCUSSION THREAD

**Discussion by Newer1107** [prasanthrangan/hyprdots#177] [Ideas]
Title, here's an example:
![image](https://github.com/prasanthrangan/hyprdots/assets/116626403/c4323dd1-f14c-4737-9c96-f267e8dbeede)

**Comment by prasanthrangan** [comment]
is this eww widget?

**Comment by Newer1107** [comment]
not sure, got this from Google

**Comment by Newer1107** [comment]
KDE also has something similar to this which looks cool
